src/pages/_app.tsx

```diff
@@ -32,7 +33,7 @@ export default class App extends NextApp {
         <SocialProfileJsonLd
           type="Organization"
           name="egghead.io"
-          url={useURL()}
+          url={`${process.env.NEXT_PUBLIC_DEPLOYMENT_URL}`}
```

Comment:
this was what React deemed a hook in a class component, though the actual useUrl isn't really a hook hook, but it is based on a naming convention.